### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by flanger001** [Houdini/two_factor_authentication#193]
The last commit to master was 2019-07-25 and there are lots of open PRs that haven't been acknowledged, https://github.com/Houdini/two_factor_authentication/pull/188 in particular. Additionally there is no activity in the Gitter chat. Is this project dead? Are you accepting new maintainers?

**Comment by steobrien** [comment]
@Houdini a quick ping on this – thanks for your work maintaining this library so far. Before submitting a PR to fix a bug I've found, I want to make sure the project is active and accepting contributions?

**Comment by anasteele206** [comment]
I would be open to taking ownership of this repo to oversee further development.

**Comment by jherdman** [comment]
@Houdini sorry to pile on in this PR, but it'd be helpful to know the state of this library.

**Comment by flanger001** [comment]
I think the best option here is going to be forking unfortunately.

**Comment by flanger001** [comment]
I will be happy to fork this into a new organization and give people maintainer access on it.

**Comment by sudhindra** [comment]
@flanger001  Can you please let me know if any forked library supports rails 6.1, happy to contribute 
@Houdini

**Comment by jasonperrone** [comment]
@sudhindra I opened a PR a while ago for the Rails 6 stuff. You could fork your own project and steal the PRs you need.

**Comment by jasonperrone** [comment]
Sorry, it wasn't me who did that PR. Losing track because I am finding I have to fix a lot of gems lately for either Rails 6.1 compatibility or Ruby 3 compatibility https://github.com/Houdini/two_factor_authentication/pull/204

**Comment by kengreeff** [comment]
@Houdini do you think we could add a few more people to this as maintainers?

**Comment by baarkerlounger** [comment]
Have forked this here https://github.com/baarkerlounger/two_factor_authentication for the time being (happy to create PRs from that here or have commits cherry-picked if that helps, if this repo gets going again). Works in testing so far with Rails 7 and Turbo, and specs run again.

**Comment by strzibny** [comment]
@baarkerlounger thanks a lot!

**Comment by baarkerlounger** [comment]
Seems like no movement in this repo still. Given that, would anyone else like to help maintain a fork :point_up:? What do people think about publishing to RubyGems? (Would need a new name presumably).

**Comment by strzibny** [comment]
I actually decided to fix https://github.com/wmlele/devise-otp. It now works for Rails 7 and the only thing missing is updating README and doing a proper release. So if someone needs maintained and up to date OTP solution join us :). I got commit and RubyGems push rights from the original dev, so no blockers.

**Comment by baarkerlounger** [comment]
@strzibny any major advantages to that project over this one?

**Comment by strzibny** [comment]
Not really, although I am not sure if `two_factor_authentication` ships with JavaScript QR code library to show the image. I like it since it's simple, small codebase, and we used it at work (I maintained Rails 5 fork before going to fix it for Rails 7).

**Comment by flanger001** [comment]
Thanks @strzibny (love Deployment From Scratch btw) and @baarkerlounger for taking care of this stuff - I actually did end up forking it into a new org myself: https://github.com/devise2fa/two_factor_authentication - but I never did anything with it afterwards because I felt the whole thing of cutting a gem was going to be a pain. However, if anyone wants to get in and actually do it, I'll happily turn over the org.

**Comment by strzibny** [comment]
Thanks @flanger001!

Btw I just released devise-otp 0.3.0 which I use with Rails 7 and Hotwire.

**Comment by baarkerlounger** [comment]
@flanger001 I submitted a pull request to your repo with all the changes I made on my fork. Any interest in merging and releasing as a new gem still? (Happy to be added as a maintainer also), or are you moving on to devise-otp rather?

**Comment by flanger001** [comment]
@baarkerlounger I'm very sorry! I just got this and I will respond to you there!
